README.md

```diff
@@ -1 +1,67 @@
-# dbqa
\ No newline at end of file
+# Double-bracket quantum algorithms
+
+Qibo features a model implementing double-bracke quantum algorithms (DBQAs) which are helpful for approximating eigenstates based on the ability to run the evolution under the input Hamiltonian.
+
+More specifically, given a Hamiltonian $H_0$, how can we find a circuit which after applying to the reference state (usually $|0\rangle^{\otimes L}$ for $L$ qubits) will approximate an eigenstate?
+
+A standard way is to run variational quantum circuits. For example, Qibo already features  the `VQE` model [2] which provides the implementation of the variational quantum eigensolver framework.
+DBQAs allow to go beyond VQE in that they take a different approach to compiling the quantum circuit approximating the eigenstate.
+
+## What is the unitary of DBQA?
+
+Given $H_0$ we begin by assuming that we were given a diagonal and hermitian operator $D_0$ and a time $s_0$.
+The `dbi` module provides numerical strategies for selecting them.
+For any such choice we define the bracket
+$$ W_0 = [D_0, H_0]$$
+and the double-bracket rotation (DBR) of the input Hamiltonian to time $s$
+$$H_0(s) = e^{sW} H e^{- s W}$$
```

Comment:
```suggestion
$$H_1(s) = e^{sW} H_0 e^{- s W}$$
```
For sure on the right we need $H_0$, I'm not sure if on the left we should have $H_1$ or something else.